packages/framework/lib/modules/index.ts

```diff
@@ -71,7 +71,7 @@ export function addPostbackRule<U extends User<any>>(
 
 export function addTextRule<U extends User<any>>(
     module: Module<U>,
-    action: (user: U, ...args: any[]) => Promise<any>,
+    action: (user: U, param: object) => Promise<any>,
```

Comment:
Oh I didn't see this... no keep it as `...args: any[]` please

Reply:
I've change cause of this discussion  https://github.com/chrispanag/ebony/pull/29#discussion_r756700797

Reply:
Remind me: Does this affect only the Viber version or Messenger ones as well?

Reply:
This affects both. Cause we do not only give plain text back as a parameter to the action function. 
We can still check if the only parameter is text and pass it as string and no an {text:string} but I don't think that this is the cleanest solution.

Reply:
cool make it as this: 

`(user: U, message?: Record<string, any>, ...args: any[])`

Reply:
The main goal here is to leave the list of params (...args: any[]) which nobody knows the order, and go to a clean object param with attributes name both in viber and messenger

Reply:
perfect that's great!